packages/sanity-plugin-nacelle-input/src/hooks/useHailFrequency.js

```diff
@@ -0,0 +1,97 @@
+import { useState, useEffect } from 'react'
```

Comment:
For clarity (of anyone who explores this project's code), might be worth renaming this hook to `useNacelleStorefront`.

Reply:
All set